src/worker.rs

```diff
@@ -69,6 +104,8 @@ fn list_files(logger: &Logger, name: &str, recursive: bool) -> Result<Vec<File>,
         .arg(format!("{}/{}/trunk", ORGANIZATION_ROOT, name))
         .output();
 
+    let query_name = name;
```

Comment:
This isn't necessary. Cloning below might fix your issue.

Reply:
Doesn't name get turned into None later on though?